core/primitives-core/src/version.rs

```diff
@@ -120,6 +120,7 @@ pub enum ProtocolFeature {
     RejectBlocksWithOutdatedProtocolVersions,
     #[cfg(feature = "protocol_feature_simple_nightshade_v2")]
     SimpleNightshadeV2,
+    PostStateRoot,
```

Comment:
Pretty sure rustc will complain about unused enum variants. Either mark it as an exception, or maybe just wait with introducing it until you have your first PR that actually needs it.

Reply:
Seems like rustc is OK with it 🤷

Reply:
Oh right because it's `pub` it may be used outside the crate! Well, then I'm even more confused why we bother with hiding even the enum variants behind compilation flags, seems completely pointless to me. :thinking: